### ISSUE TITLE
Missing build prerequisites in README.md

### ISSUE THREAD

**Issue by stefanct** [roddhjav/pass-import#69]
To build (on Debian) `python3-setuptools` must be installed or else:

```
$ make
Traceback (most recent call last):
  File "setup.py", line 4, in <module>
    from setuptools import setup
ImportError: No module named 'setuptools'

```

Additionally due `fatal error: Python.h: No such file or directory` using coverage is skipped(?) if `libpython3.5-dev` is not installed (best done by installing `python3-dev`). But it apparently builds without it automatically. It is also not retried if calling `make` again but I have not tried to find out why/how the Makefile works or if the egg magic is saving everything.
Also `s/builded/built/` in `Makefile` ;)

**Comment by stefanct** [comment]
@chrisswanda I assume that's because you are using a python2 `pip` but python3 `python` environment (or vice versa) and not related to `pass-import`.

**Comment by roddhjav** [comment]
Thank for this feedbacks. This is now fixed in df6d3a95990ec2d16e40a649cc07599eb58513ca
